card/mod/admin/spec/set/self/admin_info_spec.rb

```diff
@@ -0,0 +1,10 @@
+# -*- encoding : utf-8 -*-
+
+describe Card::Set::Self::AdminInfo do
```

Comment:
`describe` on the top level still works but it's a deprecated pattern that RSpec pollutes the global namespace like that. The better way is `RSpec.describe`.
IDE likes Rubymine also expect it that way.

fyi you can generate these files with `bundle exec decko generate card:set admin self admin_info --spec-only`

Reply:
`RSpec.describe Card::Set::Self::AdminInfo do....`
Yes?

Reply:
yes.

Reply:
Yes.